src/main/java/uk/gov/dvsa/view/cvs/CvsOdometerReadingFormatter.java

```diff
@@ -9,8 +9,9 @@ public class CvsOdometerReadingFormatter {
 
     public static final String MILES = "miles";
     public static final String MILES_WELSH = "milltiroedd";
-
     private static final String MILES_UNIT = "mi";
+    public static final String KILOMETERS = "kilometres";
+    public static final String KILOMETERS_WELSH = "cilomedr";
```

Comment:
Should this be "cilometrau" as per Emyr/Joe's comments on the ticket?

Reply:
This change has now been made and is being deployed as per Emyr/Joe comments